engine/MapUpdateSystem.ts

```diff
@@ -24,7 +23,7 @@ const $normalScaleViewerPositionDelta = new Array(2) as [number, number]
 const $previousViewerPosition = new Vector3()
 const $previousMapCenterPoint: LongLat = Array(2)
 
-export default async function MapUpdateSystem(world: World): Promise<System> {
+export default async function MapUpdateSystem(): Promise<any> {
```

Comment:
systems now use `defineSystem` which takes `execute` as an argument. any async logic should be done in a `reactor` on the system defintion